packages/askui-nodejs/src/lib/ui-controller-linux.ts

```diff
@@ -1,10 +1,40 @@
 import { exec } from 'child_process';
+import { promisify } from 'util';
 import { UiControllerFacade } from './ui-controller-facade';
 import { logger } from './logger';
+import { WaylandError } from './wayland-error';
+import { LibfuseError } from './libfuse-error';
 
 export class UiControllerLinux extends UiControllerFacade {
   // eslint-disable-next-line class-methods-use-this
   protected override makeBinaryExecutable(): void {
     exec(`chmod +x "${this.binaryPath}"`, (_exception, stdout) => logger.debug(stdout));
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  protected override async preStartChecks(): Promise<void> {
```

Comment:
Don't like the name as it seems like something static but it is a function. What do you think about `runPreStartChecks()`?